features/user_can_view_received_messages.feature

```diff
@@ -0,0 +1,18 @@
+Feature: User can view sent message
+    As a User 
+    In order to read my messages
+    I want to see them in my inbox
+
+    Background:
+        Given the following user exist:
+            | name   | email          | password | id |
+            | dummy1 | user1@mail.com | password | 1  |
+            | dummy2 | user2@mail.com | password | 2  |
+        And I am logged in as "dummy1"
+        And the inbox has content
+
+    Scenario: To view sent message
+        When I am in the inbox
+        Then I should see "Til the end of time"
+        And I should see "Subject: To dear user1"
+        And I should see "dummy1"
```

Comment:
Could you explain this assertion?

Reply:
In an earlier feature test the following background step was created:
`And the inbox has content`
Implemented as
`
Given("the inbox has content") do
    user1 = User.find(1) . 
    user2 = User.find(2)
    user1.send_message(user2, "For ever yours", "To dear user2")
    user2.send_message(user1, "Til the end of time", "To dear user1")
end
`
Since we are logged in as user1 (dummy1, with id:1) we want to look for the message sent from user 2 to user1, which has body "Til the end of time" and subject "To dear user1".

Reply:
Tried to format the code snippet but it's stupid

Reply:
Okay, I understand the tests at line 16 and 17. But I don't really understand why the user should see "dummy1"(the test at line 18).

Reply:
You are correct, it is only on the 'Sent' page someone thought it was a better idea to show the name of the sender(current user) instead of the recipient of the sent message. In the inbox it should show the sender of the received message. The test falsely passes because it finds "dummy1" in the header. Test has been updated.